AirCasting/Settings/BackendSettingsView.swift

```diff
@@ -13,20 +13,22 @@ struct BackendSettingsView: View {
     @State private var pathText: String = ""
     @State private var portText: String = ""
     @State private var url: URL?
+    @State private var urlWithoutPort: URL?
+    @State private var port: URL?
```

Comment:
I'm not sure why did you separate those? There can be single URL for this and port can be changed using `URLComponents`:
```Swift
let url = URL(string: "http://www.google.com")!
var components = URLComponents(url: url, resolvingAgainstBaseURL: false)!
components.port = 66
let newUrl = components.url
```